xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-livetable/src/main/java/org/xwiki/livedata/internal/livetable/LiveTableLiveDataPropertyStore.java

```diff
@@ -147,7 +162,14 @@ private LiveDataPropertyDescriptor getLiveDataPropertyDescriptor(PropertyClass x
             descriptor.setFilter(new FilterDescriptor("list"));
             descriptor.getFilter().addOperator("empty", null);
             if (xproperty instanceof LevelsClass) {
-                descriptor.getFilter().setParameter("options", ((LevelsClass) xproperty).getList(xcontext));
+                // We need to provide a list of maps of value / labels so that selectize can interpret them.
+                descriptor.getFilter().setParameter("options", ((LevelsClass) xproperty).getList(xcontext)
+                    .stream()
+                    .map(item -> Map.of(
+                        "value", item,
+                        "label", getRightTranslationWithFallback(item)
+                    ))
+                    .toList());
```

Comment:
I don't see fix versions on the Jira issue. Are all the branches you plan to backport to supporting `toList` ?

Reply:
Hmmm I'm planning to backport it on 15.10.x and indeed it might not be good there.